src/pepr/policies/index.ts

```diff
@@ -1,12 +1,16 @@
 // Various validation actions for Kubernetes resources from Big Bang
-import { K8s, Log } from "pepr";
+import { K8s } from "pepr";
+import { setupLogger } from "../logger";
 import { ExemptionStore } from "../operator/controllers/exemptions/exemption-store";
 import { processExemptions } from "../operator/controllers/exemptions/exemptions";
 import { Matcher, Policy, UDSExemption } from "../operator/crd";
 import "./networking";
 import "./security";
 import "./storage";
 
+// configure subproject logger
+const log = setupLogger("policies");
```

Comment:
It definitely matches the structure for the naming we chose but watching the logs around the test exemptions they end up looking like this:
```
{"level":20,"time":1720454522447,"pid":17,"hostname":"pepr-uds-core-774bfd4b86-v4lt7","component":"policies","msg":"Processing exemption podinfo3, watch phase: ADDED"}
{"level":20,"time":1720454522447,"pid":17,"hostname":"pepr-uds-core-774bfd4b86-v4lt7","component":"operator.exemptions","msg":"No existing exemption for owner 84cb0536-7a78-487e-8b3d-d872c9a21a9a"}
{"level":20,"time":1720454522447,"pid":17,"hostname":"pepr-uds-core-774bfd4b86-v4lt7","component":"operator.exemptions","msg":"Added exemption to DropAllCapabilities: {\"kind\":\"pod\",\"name\":\"^podinfo.*\",\"namespace\":\"podinfo\"}"}
...
{"level":20,"time":1720454523025,"pid":17,"hostname":"pepr-uds-core-774bfd4b86-v4lt7","component":"policies.exemptions","msg":"Checking podinfo-dd8b569b- against DropAllCapabilities exemptions: [{\"kind\":\"pod\",\"name\":\"^neuvector-enforcer-pod.*\",\"namespace\":\"neuvector\",\"owner\":\"c114993e-6004-48ff-8b6a-dfa7617c8966\"},{\"kind\":\"pod\",\"name\":\"^neuvector-controller-pod.*\",\"namespace\":\"neuvector\",\"owner\":\"c114993e-6004-48ff-8b6a-dfa7617c8966\"},{\"kind\":\"pod\",\"name\":\"^podinfo.*\",\"namespace\":\"podinfo\",\"owner\":\"84cb0536-7a78-487e-8b3d-d872c9a21a9a\"}]"}
{"level":30,"time":1720454523025,"pid":17,"hostname":"pepr-uds-core-774bfd4b86-v4lt7","component":"policies.exemptions","msg":"podinfo-dd8b569b- is exempt from DropAllCapabilities"}
...
{"level":20,"time":1720454530256,"pid":17,"hostname":"pepr-uds-core-774bfd4b86-v4lt7","component":"policies","msg":"Processing exemption podinfo3, watch phase: DELETED"}
{"level":20,"time":1720454530256,"pid":17,"hostname":"pepr-uds-core-774bfd4b86-v4lt7","component":"operator.exemptions","msg":"Removed all policy exemptions for 84cb0536-7a78-487e-8b3d-d872c9a21a9a"}
```

Basically we end up with a mix of logs labelled as `policies`, `operator.exemptions`, and `policies.exemptions`. I think this is okay if we want to stick with the rigid naming based on location (maybe it's a good sign we need to move around some of our code 🤷)? Just thought I'd call it out as a bit of a weird result, if I wanted to easily filter on all exemption logs that wouldn't be the easiest to do?

Reply:
I think for now this is okay - we'll probably revisit some of the metadata around these logs in the future and can evaluate moving logs around/changing names at that point. Thinking through the existing 3 log types for exemptions:
- `policies.exemptions`: these are the actual validation checks against the existing exemptions
- `operator.exemptions`: these are the `Exemption` CR lifecycle handling, adding/removing exemptions from our "store"
- `policies`: these are related to the `Exemption` CR watch setup/events

Really it's just that `policies` one that feels a bit out of place, might be good to re-evaluate in the future if this code should move to the operator folders since it's closer to those pieces. For now don't want to propose any changes since this is primarily meant to reduce the verbosity of pepr's logs, with the `component` addition being an added bonus and start towards better structured core logs.